x/emissions/proto/emissions/v1/types.proto

```diff
@@ -269,6 +271,13 @@ message Scores {
   repeated Score scores = 1;
 }
 
+/// AVERAGE REWARD
+
+message AvgReward {
```

Comment:
should be consistent -- you say "AverageReward" everywhere but here, where you use shorthand

Reply:
fixed this myself